R/Map_Functions.R

```diff
@@ -468,7 +468,14 @@ plot_GCAM <- function(mapdata, col = NULL, proj = robin, extent = EXTENT_WORLD,
 
     # Generate graticule (latitude/longitude lines) and clip map to extent specified.
     grat <- gen_grat()
-    mappolys <- get_bbox_polys(dataset = mapdata)
+    
+    # Only clip polygons if using an AEA projection; otherwise this can induce distortions
+    if(proj %in% c(na_aea, ch_aea)){
```

Comment:
Ok. I agree that this is only a first-order fix that does not address what is really going on. To avoid problems with the global Antarctica maps while we solve the core issue, I propose an if-statement saying that if the projection is global, don't use the get_bbox_polys function. 
As for the get_bbox_polys function, I think what's happening is that we have data that is 180.00000 and a bounding box with 180 (those are the points I noticed that were getting clipped out). I think it may be a type issue. I don't think our map data is necessarily the problem, but I'll look into that more.